tests/Fixer/LanguageConstruct/IsNullFixerTest.php

```diff
@@ -76,6 +76,7 @@ public function testNonYodaFix()
             '<?php $b = a(a(a(b() === null) === null) === null) === null;',
             '<?php $b = \is_null(a(\is_null(a(\is_null(a(\is_null(b())))))));'
         );
+        $this->doTest('<?php if ($x === null && $y) echo "foo";', '<?php if (is_null($x) && $y) echo "foo";');
```

Comment:
This is currently the only test that fails without your proposed fixes.

The test passes without your fixes, when you disable this line;
https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/v2.2.7/src/Fixer/LanguageConstruct/IsNullFixer.php#L106

This makes me think that the `IsNullFixer` should only be changed there by adding a test before the call to `removeTrailingWhitespace`. 

The other test cases are all nice, so :+1: for adding :)